descwl_coadd/coadd.py

```diff
@@ -1331,202 +793,85 @@ def _finish_init(self):
             store_pixels=False,
         )
```

Comment:
Why is the bmask zero? Why aren't we setting the ormask to zero and using the mask in the coadd as the bmask?

Reply:
I don't think we are actually using these downstream, we use the mask attribute.

And we use that mask attribute as the ormask, so there is some consistency here.

We need to add the mask frac that you use for DES

Reply:
I mean "mask attribute of the exposure attribute"

Reply:
see issue #48

Reply:
Can we punt on this and address it in #48?

Reply:
ofc!